spec/acceptance/rds_spec.rb

```diff
@@ -92,6 +92,12 @@ def get_rds_instance(name)
       end
     end
 
+    it 'with the default subnet association (VPC-only accounts)' do
+      unless @aws.vpc_only?
+        expect(@rds_instance.db_subnet_group.db_subnet_group_name).to eq('default')
```

Comment:
Are there any expectations connected to the non-`vpc_only` case?

If yes, they should be added, if no, can we perhaps push this information to a tag or add a `pending` or `skip` as alternative to make this more clear? Also connected: is this constant throughout a test run?